guides/release/tutorial/01-orientation.md

```diff
@@ -54,6 +54,15 @@ npm: Installed dependencies
 Successfully initialized git.
 ```
 
+```shell
+$ ember server
+Could not start watchman
+Visit https://ember-cli.com/user-guide/#watchman for more info.
```

Comment:
https://github.com/chrmod/watch-detector/pull/8

Reply:
This is now merged into ember-cli master, this line should be gone with CLI 3.14-beta